test/testAssets.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const assert = require('assert');
+const vfs = require('vinyl-fs');
+const sinon = require('sinon');
+
+const assets = require('../lib/assets');
+
+describe('assets', function() {
+  it('Runs the vfs pipeline', async function() {
```

Comment:
Oops. I forgot that we still support Node 6 (and run our tests against it), which does not have support for `async/await`. If it was important we could drop support for Node 6, but in this case I think we might as well just use other syntax options.